### PR TITLE
Added meta tags, pulled from Strapi backend

### LINKED ISSUE #124: Add SEO optimizations

### PR DESCRIPTION
**Pull request by LemonFace0309**
closed #124

read commit messages for more info.

image

Added dynamic meta tags for articles and improved scrolling experience (try clicking on media or contact).

### CODE REVIEW

File: pages/msmr-articles/[slug].js [hunk 1]
```diff
@@ -28,8 +29,29 @@ import styles from '../../styles/Article.module.css'
...
   const shareUrl = `${process.env.NEXT_PUBLIC_HOST}/featured-artciles/${router.query.slug}`
+
   return (
     <>
+      <Head>
+        <title>{article.title}</title>
+        <meta
+          name="description"
+          content={article.description || article.title}
+        ></meta>
```
Comment (jimmyygit) [inline pages/msmr-articles/[slug].js:40]: Should we add a keywords tag? like psychology and blah

File: pages/msmr-articles/[slug].js [hunk 2]
```diff
@@ -81,7 +103,7 @@ function Article({ article }) {
           </Col>
         </Row>
         <Row className="mt-4 py-4">
-          <ReactMarkdown children={article.content} />
+          <ReactMarkdown allowDangerousHtml={true} children={article.content} />
```
Comment (jimmyygit) [inline pages/msmr-articles/[slug].js:106]: what is considered dangerous lol

**Review by jimmyygit — APPROVED** [review]
Rock Solid :)

File: pages/msmr-articles/[slug].js (hunk 2 above)
Comment (LemonFace0309) [reply pages/msmr-articles/[slug].js:106]: inline html

**Pull request merged**